docs/learn-collator.md

```diff
@@ -28,19 +28,34 @@ the censorship problem is solved with having just one honest collator.
 
 ### XCMP
 
-Collators are a key element of [XCMP (Cross-Chain Message Passing)](learn-crosschain). By being
+Collators are a key element of [XCMP (Cross-Chain Message Passing)](Due to the way the [Phragmen algorithm](learn-phragmen) generates the solution set, there _may_
+exist a minimum number of DOT needed to nominate with in order to receive staking
+rewards due to the fact that the solution set needs to be both processed and stored within
+a single block: [a target of 6 seconds on Polkadot](faq#what-is-the-block-time-of-the-relay-chain).
+
+A [runtime upgrade](build-protocol-info#runtime-upgrades) moved the logic to process the rewards
+for nominators to an [off-chain worker](learn-phragmen#off-chain-phragmén), and now the minimum
+DOT required is only limited by the [existential deposit](build-protocol-info#existential-deposit).
+If the number of nominators increases dramatically, the solution will be pruned and nominators with
+the least amount of stake will once again be pruned and receive no staking rewards to ensure that 
+block times remain on target.
+
+See the [Election Solution Set](learn-nominator#the-election-solution-set) section on the
+nominator page for more details.
+). By being
```

Comment:
HUH?

Reply:
https://github.com/w3f/polkadot-wiki/pull/2030 
we can drop in in this PR, wrong branch I committed too it seems. 

you reverted it. Have a better thing to use here?